apps/docs/src/content/docs/reference/default.mdx

```diff
@@ -0,0 +1,234 @@
+---
+title: Default task
+description: the task function
+---
+
+import { LinkCard, Tabs, TabItem, Aside } from '@astrojs/starlight/components';
+
+This is the way to create a default task.
+
+There's no concurrency management in this task and every `perform` will be executed immediately.
+
+## Usage
+
+To specify a default task, you can either use the dot notation or the options notation.
+
+<Tabs syncKey="notation">
+
+<TabItem label="Dot notation">
+
+```svelte
+<script lang="ts">
+	import { task } from '@sheepdog/svelte';
+
+	const myTask = task(async () => {
+		// your code
+	});
+</script>
```

Comment:
What do you mean by "dot notation" in this case? The only difference I see between both blocks is the option object in the second one. Is there something missing?

Reply:
Yeah this is kinda strange but it's only for the default task...if you see the other task the dot notation is `task.enqueue()`

Reply:
Good idea...i'll add a small message on top

Reply:
@BlueCutOfficial do you mind jumping on a quick huddle to help me with the best phrasing for this?